### ISSUE TITLE
Quick concerns

### ISSUE THREAD

**Issue by RohanAlexander** [eeeee-cmd/US_Election#1]
- Be specific about the finding in the overview of the README.
- Commit messages could benefit from being more detailed.
- Simulation - why not include some interaction between the variables?
- Testing - why not consider testthat?
- Use a subtitle to convey the main finding.
- Abstract - be specific about the finding.
- Introduction is too wordy. Be concise and specific.
- Mode results - don't forget to format these tables nicely and not include raw R output.
- Captions need more detail.
- Figures 5 and 6 x axis labels are hard to read.
- Appendices need a lot more detail.
- R packages capitalization is wrong - see example in starter folder for how to fix.
- R NOT CITED
- DATA NOT CITED.

**Comment by RayanAlim** [comment]
Addressed review as follows:
- Ensured our commit messages were appropriate
- Simulation- added interaction between variables 
- Testing: Used testthat 
- Used subtitle to convey the main finding - closer race but Trump, trumps Harris
- Specific findings added to abstract
- Tightened the introduction, removed filler words and jargon.
- Removed R output from paper
- Used Kable to format results nicely
- Added detail caption and cross-referencing 
- Changed figure 5 from unclear boxplots to viloin plots that show distribution of just Trump, aHarris and third for all candidates combined.
- Cited R and the data
- Fixed R package capitalization
- Added more detail to the appendix.

Thanks for the helpful review!
